README.md

```diff
@@ -1,70 +1,81 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# The Spoonify Recipe App.
```

Comment:
- [ ] This is the `Math-Magicians` project. You have only changed the screenshot. Please update your Readme file to match the current project. For example, project title, description, Github link, etc. Kindly remove the `The Spoonify Recipe` app live link and video explanation.